estate/models/estate_property.py

```diff
@@ -127,3 +128,34 @@ def _onchange_garden(self):
         else:
             self.garden_area = 0
             self.garden_orientation = False
+
+    # `self` must be a single "estate.property" record
+    def _check_availability(self, error_msg):
```

Comment:
add self.ensure_one()

Reply:
Too add docstring

Reply:
Thanks for the review! I didn't know of that interface.